### DISCUSSION TITLE
Welcome to onetagger Discussions!

### DISCUSSION THREAD

**Discussion by Marekkon5** [Marekkon5/onetagger#101] [Announcements]
##  Welcome!
Feel free to ask any questions, report bugs or issues and suggest improvements here.

**Comment by Archaleas** [comment]
Hi, Any chance of releasing an android app?

I obsessively organize and curate my music from one device my phone and being able to do this on the with one tagger on the go would be amazing.

**Reply by Marekkon5** [reply]
Since this is written in Rust - porting Auto Tagger, Audio Features, and Renamer shouldn't be that hard. Quick Tag and Tag Editor would be more problematic / I wouldn't even consider doing it. However I don't think it's worth it. 
- There are some good apps such as Automatag, which were designed with mobile use in mind, and work better on mobile.
- OneTagger was designed for desktop - it would feel too advanced and complex on phone.

Let me know what features do you have in mind, or if the alternatives are enough for you. Thanks

**Comment by Archaleas** [comment]
Thank you for your reply,

Juut for a bit of background info in my workflow as this may help in your
response,

I obsessively categorise and organise my music manually on folders stored
on my phone's 1tb SD Card.

Information that I would love is *beatport *genres added to each songs tag
info, that would cutout the step of me having to google  "[song title] +
beatport" search for the genre, type it into the song tag info and then
transfer that song to the appropriate local folder.

Much appreciated

**Reply by Marekkon5** [reply]
Understandable, I will be away now for a week, will look at it afterwards. Thanks

**Comment by peanutlasko** [comment]
Hey @Marekkon5 , trying to workout which port the --server --expose runs on. Seems like its 36913 ? What's 36912 used for?

I'm using a reverse proxy which forces HTTPS, but it seems like when I do this OneTagger only responds over HTTP not HTTPS.

**Reply by Marekkon5** [reply]
36912 is the Websocket server, which is used for all of the backend-frontend communication. 36913 is just the HTTP server. You have to forward both for 1T to work properly.

**Comment by ijesaboy** [comment]
Greetings @Marekkon5 & thanks for the gift of this onetagger project.

Any chance you'll support .opus or .ogg file formats? 

I would currently have to transcode to one of your supported file types and back given opus and ogg are my preferred formats. I can achieve this with scripts outside of onetagger but this doesn't seem ideal. 

If you think there's a way to integrate opus/ogg support seamlessly and can point me in the right direction i'd very much appreciate it. i'm also happy help if i can. 

with gratitude.

**Reply by Marekkon5** [reply]
Hello, adding ogg shouldn't be hard as it uses vorbis comments (same thing as FLAC). However 1T is currently undertaking big update to use Vue3+TS, so it might take a while until I add it. However you can create an Github Issue, so we can get reminded, because we use them as feature/issue tracker. Thank you

**Reply by ijesaboy** [reply]
Thanks for your consideration and responsiveness. I've posted a Feature Request #134 as an issue and will plan to send a reminder after the big Vue3+TS update. BTW, good luck with that push -- I'm eager to see what it becomes. 

with Gratitude

**Comment by flovanc** [comment]
hello, for people having huge collection of track , there is a way to tag only new arrived files inside folders ?

**Comment by bascurtiz** [comment]
@flovanc 
Yep!
Just make sure _Skip already tagged tracks_ and _Include subfolders_ is enabled in Advanced section:

![image](https://user-images.githubusercontent.com/34075088/205961883-a1633c5b-7331-4ac2-84b4-039ddbf15fe4.png)
